app/CentreUser.php

```diff
@@ -110,7 +122,7 @@ protected function homeCentres()
     /**
      * Get the relevant centres for this CentreUser, accounting for it's role
      *
-     * @return \Illuminate\Database\Eloquent\Collection|\Illuminate\Support\Collection|static[]
+     * @return Collection|\Illuminate\Support\Collection|static[]
```

Comment:
if we're using `use` at the top of the file consistently, should we `use .. as ..` the base collection?